armory/metrics/task.py

```diff
@@ -818,6 +828,107 @@ def object_detection_mAP(y_list, y_pred_list, iou_threshold=0.5, class_list=None
     return np.fromiter(ap_per_class.values(), dtype=float).mean()
 
 
+def armory_to_tide_ground_truth(y_dict, image_id, data_ground_truth):
+    for y in [dict(zip(y_dict, t)) for t in zip(*y_dict.values())]:
```

Comment:
I really don't understand what this line is doing. Is there a way to simplify it?

Reply:
`y_dict` is a dictionary with values of list type, which is converted to a list of dictionaries to be iterated through to add to the `TIDE` data type via `add_detection` or `add_ground_truth`

Reply:
resolved with [851cfcc](https://github.com/twosixlabs/armory/pull/1762/commits/851cfcc51a72d714b1781d9dcf0de4ac81c52234)